pom.xml

```diff
@@ -43,8 +43,8 @@
   </scm>
   <developers>
     <developer>
-      <id>gabrielasmeria</id>
-      <name>Gabriela Smeria</name>
+      <id>acotiuga</id>
+      <name>Alexandru Cotiuga</name>
```

Comment:
You should not remove a past developer, unless there was a copy/paste mistake in the original pom. Instead, append a new `<developer>` tag.